File: 02-Prueba/05-Haskell/HolaMundo.hs
{-
    Programa que muestra el mensaje "Hola Mundo" en la pantalla
    Autor: Julián Esteban Gutiérrez Posada
    Fecha: Febrero 2025
    Licencia: GNU GLP v3
-}

main :: IO()
main = do
  mostrarMensaje "Hola Mundo"


mostrarMensaje :: String -> IO ()
mostrarMensaje mensaje = do
  putStr mensaje

File: 02-Funcion/04-Javascript/HolaMundo.js
/*
  Programa que muestra el mensaje "Hola Mundo" en la pantalla
  Autor: Julián Esteban Gutiérrez Posada
  Fecha: Enero 2025
  Licencia: GNU GPL v3
*/

function main() {
  mostrarMensaje("Hola Mundo\n");
}

function mostrarMensaje(mensaje) {
  document.getElementById("salida").value = mensaje;
}